.travis.yml

```diff
@@ -1,3 +1,4 @@
 language: scala
 scala:
-   - 2.11.2
+   - 2.11.12
+   - 2.10.5
```

Comment:
If 2.10 causes problems for 2.12 or 2.13 I'm happy to drop it

Reply:
It think it's still OK. It just has Specs2 and ScalaCheck version issue.